static/js/util/profile_edit.js

```diff
@@ -62,13 +63,17 @@ export function boundRadioGroupField(keySet: string[], label: string, options: O
         labelStyle={styles.labelStyle}
         value={obj.value}
         label={label}
+        aria-labelledby={labelId}
```

Comment:
It's better to use a `<label>` element rather than `aria-labelledby` if possible. You can nest the `<input>` element inside of the `<label>` element, or use the `for` attribute on the label to point to the element ID that is being labelled. Using `<label>` also makes the browser automatically understand that when the user clicks on the label text, the `<input>` field should be selected -- although it looks like Material UI is manually recreating that behavior using Javascript.